CONTRIBUTING.md

```diff
@@ -87,113 +85,154 @@ Usage: make <target>
 ...output...
 ```
 
-### Install tools
+### Install required tools
 
-Makefile allow you to install needed tools:
+Makefile allow you to install needed development tools (e.g. the linter):
 ```shell
 $ make install.tools
 ```
 
-### Building binaries and test your changes
+### Building binaries
 
-To test your changes do `make binaries` to generate your binaries.
+To build Podman binaries, you can run `make binaries`.
+Built binaries will be placed in the `bin/` directory.
+You can manually test to verify that Podman is working by running the binaries.
 
-Your binaries are created inside the `bin/` directory and you can test your changes:
-```shell
-$ bin/podman --help
-bin/podman --help
-NAME:
-   podman - manage pods and images
-
-USAGE:
-   podman [global options] command [command options] [arguments...]
-
-VERSION:
-   1.0.1-dev
-
-COMMANDS:
-     attach           Attach to a running container
-     build            Build an image using instructions from Dockerfiles
-     commit           Create new image based on the changed container
-     container        Manage Containers
-     cp               Copy files/folders between a container and the local filesystem
-```
+For further reading about branching [you can read this document](https://herve.beraud.io/containers/linux/podman/isolate/environment/2019/02/06/how-to-hack-on-podman.html).
 
-Well, you can now create your own branch, apply changes on it, and then submitting your pull request.
+### Building docs
 
-For further reading about branching [you can read this document](https://herve.beraud.io/containers/linux/podman/isolate/environment/2019/02/06/how-to-hack-on-podman.html).
+To build Podman's manpages, you can run `make docs`.
+Built documentation will be placed in the `docs/build/man` directory.
+Markdown versions can be viewed in the `docs/source/markdown` directory.
+Files suffixed with `.in` are preliminary versions that are compiled into the final markdown files.
+
+## Libraries
+
+Podman uses a large amount of vendored library code, contained in the `vendor/` directory.
+This code is included in the Podman repository, but is actually maintained elsewhere.
+Pull requests that change the vendor/ directory directly will not be accepted.
+Instead, changes should be submitted to the original package (defined by the path in `vendor/`; for example, `vendor/github.com/containers/storage` is the [containers/storage library](https://github.com/containers/storage/).
+Once the changes have been merged into the original package, Podman's vendor directory can be updated by using `go get` on the appropriate version of the package, then running `make vendor-in-container`.
```

Comment:
`make vendor` or `make vendor-in-container`, I really do not see why a container is needed for that